third-party/linshare/src/main/java/org/apache/james/linshare/LinshareConfiguration.java

```diff
@@ -30,67 +30,82 @@
 
 public class LinshareConfiguration {
 
-    public static final String URL = "blob.export.linshare.url";
-    public static final String TOKEN = "blob.export.linshare.token";
+    public static final String URL_PROPERTY = "blob.export.linshare.url";
+    public static final String USER_NAME_PROPERTY = "blob.export.linshare.user.name";
+    public static final String PASSWORD_PROPERTY = "blob.export.linshare.user.password";
 
     public static class Builder {
         @FunctionalInterface
         public interface RequireUrl {
-            RequireAuthorizationToken url(URL url);
+            RequireBasicAuthorization url(URL url);
 
-            default RequireAuthorizationToken urlAsString(String url) throws MalformedURLException {
+            default RequireBasicAuthorization urlAsString(String url) throws MalformedURLException {
                 return url(new URL(url));
             }
         }
 
-        public interface RequireAuthorizationToken {
-            ReadyToBuild authorizationToken(AuthorizationToken token);
+        public interface RequireBasicAuthorization {
+            ReadyToBuild basicAuthorization(String username, String password);
```

Comment:
It's not a username but a uuid, and it should be strongly typed (UUID?)

Reply:
Yes, it will be UUID.

Reply:
Im afraid we still needed to use String UUID instead.Because **LinshareConfiguration** are using for both **user** and **technicalAccount** for authentication in testing and they used different field.

Reply:
You can provide a default method that takes a UUID, or we could split the delegation and user API?

Reply:
The problem is `username` is in different format with `UUID` and we are using for both of them.
Yes we should separate another class for testing only.